tests/test_apply.py

```diff
@@ -42,132 +42,110 @@
 )
 
 
-single_qubit = [
-    ("PauliX", X),
-    ("PauliY", Y),
-    ("PauliZ", Z),
-    ("Hadamard", H),
-    ("S", S),
-    ("T", T),
-    ("S.inv", S.conj().T),
-    ("T.inv", T.conj().T),
+single_qubit_operations = [
+    qml.PauliX,
+    qml.PauliY,
+    qml.PauliZ,
+    qml.Hadamard,
+    qml.S,
+    qml.T
 ]
 
-
-single_qubit_param = [("PhaseShift", phase_shift), ("RX", rx), ("RY", ry), ("RZ", rz)]
-two_qubit = [("CNOT", CNOT), ("SWAP", SWAP), ("CZ", CZ)]
-two_qubit_param = [("CRZ", crz)]
-three_qubit = [("Toffoli", toffoli), ("CSWAP", CSWAP)]
-
+single_qubit_operations_param = [qml.PhaseShift, qml.RX, qml.RY, qml.RZ]
+two_qubit = [qml.CNOT, qml.SWAP, qml.CZ]
+two_qubit_param = [qml.CRZ]
+three_qubit = [qml.Toffoli, qml.CSWAP]
 
 @pytest.mark.parametrize("analytic", [True])
 @pytest.mark.parametrize("shots", [8192])
 @pytest.mark.usefixtures("skip_unitary")
-class TestStateApply:
-    """Test application of PennyLane operations to state simulators."""
+class TestAnalyticApply:
+    """Test application of PennyLane operations with analytic attribute set to True."""
```

Comment:
@josh146 included these new names and updates here, let me know what you think!

Reply:
Sounds much better now!